src/handleScroll.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * Scrolls the window to
+ * the provided x, y coordinates.
+ *
+ * Defaults to (0,0) if none are given.
+ *
+ * @param {number} x
+ * @param {number} y
+ *
+ * @returns {void}
+ */
+function handleScroll(x = 0, y = 0) {
```

Comment:
We may want to change the naming of `handleScroll` since `src/ScrollTo.jsx` also has a method of the same name. What about changing this to `scrollWindow`?

Reply:
👍 Updated.